main.go

```diff
@@ -19,6 +19,8 @@ var (
 	allTeams     = map[string]Team{}
 	allTeamNames = []string{}
 	cache        = map[string]CacheEntry{}
+	homeTmpl     *template.Template
```

Comment:
What if you instead used a map of the name of the template to the template object?